docs/NOTIFICATIONS.md

```diff
@@ -0,0 +1,49 @@
+# NearIt-UI for notifications
+### Heads-up notifications for in-app content
+NearIt-UI can manage and show in-app content (read more about background and foreground notifications [here](http://nearit-android.readthedocs.io/en/latest/in-app-content/))
+for you. When a user enters in a specific beacon range, an heads-up notification (on devices with Android API level 21+) will show. If the user clicks on it, the right dialog will appear and the right tracking events will be handled.
+
+![ranging_notif](notifications.gif)
+
+To enable this behaviour you have to add one line of code.
+You **MUST** add this in the `onCreate()` method of your `Application`
+
+```java
+public class YourApplication extends Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        NearITUIBindings.enableAutomaticForegroundNotifications(this);
+        //  ...
+    }
+}
+```
+
+Please note that foreground notifications will auto-dismiss after a small amount of time.
+
+### Background notifications
+
+NearIT-SDK has a built-in service that will handle every background event and show simple notifications.
```

Comment:
Tapping on those notification will deliver an Intent to your app launcher.

Reply:
done